docs/configuration/resource-recommender.rst

```diff
@@ -0,0 +1,126 @@
+Cost Savings (KRR)
+************************************************************
+
+Robustas `KRR <https://github.com/robusta-dev/krr>`_ is a CLI tool for optimizing resource allocation in Kubernetes clusters.
+It gathers pod usage data from Prometheus and recommends requests and limits for CPU and memory.
+This reduces costs and improves performance.
+By default, every instance of Robusta that's connected to the UI will run a KRR scan on startup. Further KRR scans can be triggered in the UI, and all scans can be viewed there.
+
+With or without the UI, you can configure additional scans on a :ref:`schedule <Scheduled>`.
```

Comment:
Can we add explicit details on this, just like we do for popeye. You can copy the instructions from https://github.com/robusta-dev/krr?tab=readme-ov-file#setup